example/lambdas.yml

```diff
@@ -23,30 +23,41 @@ DiscoverGranules:
   timeout: 300
   memory: 512
   source: node_modules/@cumulus/discover-granules/dist/
-  useMessageAdapter: true
+  layers:
+    - '{{cma_layer}}'
   launchInVpc: true
 DiscoverPdrs:
   handler: index.handler
   timeout: 300
   source: node_modules/@cumulus/discover-pdrs/dist/
-  useMessageAdapter: true
+  layers:
+    - '{{cma_layer}}'
   logToElasticSearch: true
   launchInVpc: true
 FakeProcessing:
   handler: index.handler
   source: node_modules/@cumulus/test-processing/dist/
-  useMessageAdapter: true
+  layers:
```

Comment:
Unnecessary?

Reply:
Not sure I understand the question here - as part of this PR, I wanted to move most of our workflows to utilizing the layer rather than the injection, and leave a test for the legacy behavior.

Reply:
I think I misread this block as being empty before. Disregard